src/interventions.c

```diff
@@ -534,12 +534,22 @@ void intervention_notify_contacts(
 			for( idx = 0; idx < n_contacts; idx++ )
 			{
 				contact = inter->individual;
-				if( contact->app_user )
-				{
-					if( inter->traceable == UNKNOWN )
-						inter->traceable = gsl_ran_bernoulli( rng, params->traceable_interaction_fraction );
-					if( inter->traceable )
+
+				// We first check if this interaction is manually traceable
+				if( inter->manual_traceable == UNKNOWN )
+					inter->manual_traceable = gsl_ran_bernoulli( rng, params->traceable_interaction_fraction );
+
+				// Then we check if this interaction is digitally traceable
+				if( inter->app_traceable == UNKNOWN )
+					inter->app_traceable = gsl_ran_bernoulli( rng, params->traceable_interaction_fraction );
+
+				// If it is manually traceable, we trigger the interventions
```

Comment:
Given the function call is currently identical for manual or app based tracing, can you consolidate in to a single if statement.

Reply:
Completed with slight changes - consolidated into single if statement, removed the extra property in the 'inter' object. Now differentiated by interaction network, and need to retain information about trace type. See other discussion on 'manual_traceable_interaction_fraction'